app/celery/utils.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Dict, Optional
```

Comment:
I had to move the parameters logic in its own module to avoid a circular import issue, as it was previously in a notifications __init__ module. This would create issues.